autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -8,128 +8,152 @@
 from scraper.models.course import generate_course_id
 from scraper.models.section import generate_meeting_id
 
-def convert_meeting_time(string_time: str):
-    """ Converts a meeting time from a string in format hhmm to datetime.
-            ex) 1245 = 12:45am. 1830 = 6:30 pm."""
+def convert_meeting_time(string_time: str) -> datetime.time:
+    """ Converts a meeting time from a string in format hhmm to datetime.time object.
+        ex) 1245 = 12:45am. 1830 = 6:30 pm.
+    """
+
     if string_time == "" or string_time is None:
```

Comment:
```suggestion
    if not string_time:
```
Since we're just checking if `string_time` is a falsy value and those are the only 2 of them for strings, we can do this instead